### PR TITLE
Add removing old sessions functionality and cron script

### PR DESCRIPTION
**Pull request by alexbagirov**
refs #7

### CODE REVIEW

File: session_cleaner.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+stamp = datetime.now()
+log.debug('current timestamp is' + str(stamp.timestamp()))
+
+# Вычитаем два часа
+stamp -= timedelta(hours = 2)
+log.debug('will remove everything older than' + str(stamp.timestamp()))
+
+# Удаляем все записи, в которых время меньше stamp
+c.execute('''DELETE FROM sessions
+             WHERE last_active < %s''', (str(stamp)))
```
Comment (illright) [inline session_cleaner.py]: `str(stamp)` is a string representation of a `datetime` object, not the timestamp. Also, you forgot a comma to make the parentheses form a tuple

File: processors.py [hunk 2]
```diff
@@ -321,6 +322,13 @@ def _next_free_dialog(self):
         c.close()
         return dialogs[-1] + 1
 
+    def _set_timestamp(self, address):
+        """Сохраняет текущую дату в last_active
+        таблицы sessions"""
```
Comment (illright) [inline processors.py:327]: Why on Earth is following general conventions that hard?

**Review by illright — CHANGES_REQUESTED** [review]

File: processors.py (hunk 2 above)
Comment (alexbagirov) [reply processors.py:327]: If you want to say something, just say it and not the shit you've just said.

File: session_cleaner.py (hunk 1 above)
Comment (alexbagirov) [reply session_cleaner.py]: So yes, we have `em stored as text and it seems to be fine.

File: processors.py (hunk 2 above)
Comment (illright) [reply processors.py:327]: Well, it's the third time my comment disappears, so fine, I'll say it again. A `.timestamp()` method is to be called on an `datetime` object with no arguments, not on a class. We don't call `str.upper('string')`, the same thing here

File: session_cleaner.py [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+stamp = datetime.now()
+log.debug('current timestamp is' + str(stamp.timestamp()))
+
+# Вычитаем два часа
+stamp -= timedelta(hours = 2)
+log.debug('will remove everything older than' + str(stamp.timestamp()))
+
+# Удаляем все записи, в которых время меньше stamp
+c.execute('''DELETE FROM sessions
+             WHERE last_active < %s''', (str(stamp), ))
```
Comment (illright) [inline session_cleaner.py]: You want `str(stamp.timestamp())` here. Don't confuse yourself with misleading variable names. Call it `date` or something

**Comment by alexbagirov** [comment]
Ready for review :tada:

**Review by illright — APPROVED** [review]

**Pull request merged**